v_3_6_0/master_template.go

```diff
@@ -2153,6 +2189,7 @@ coreos:
       -v /run/docker.sock:/run/docker.sock:rw \
       -v /usr/lib/os-release:/etc/os-release \
       -v /usr/share/ca-certificates/:/etc/ssl/certs \
+      -v /var/lib/calico/:/var/lib/calico \
```

Comment:
From 3.1 they store node name in `/var/lib/calico`, which afaik is used by CNI. So we need to mount it for kubelet